src/config.ts

```diff
@@ -127,7 +127,7 @@ const defaultConfig: IConfig = {
     },
     dataPath: "/data/storage",
     acceptInvitesFromSpace: '!noop:example.org',
-    autojoinOnlyIfManager: false,
+    autojoinOnlyIfManager: true,
```

Comment:
Could you take the opportunity to document what `autojoinOnlyIfManager` does in `IConfig`?